### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by DragonEggBedrockBreaking**
- Added github actions to build on push
- Bumped loom to 0.6
- Added a 6.8.3 gradle wrapper with an executable gradlew
- Some changes in fabric.mod.json
- Removed some useless lines in other places

### CODE REVIEW

**Comment by DragonEggBedrockBreaking** [comment]
- removed mixins file and it’s reference in fabric.mod.json

**Comment by DragonEggBedrockBreaking** [comment]
- updated loom to 0.7
- updated to gradle 7
- removed some deprecations in build.gradle.kts

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
```
Comment (toinouH) [inline gradle/wrapper/gradle-wrapper.properties]: Maybe update this to 7.1 ?

Comment (DragonEggBedrockBreaking) [reply gradle/wrapper/gradle-wrapper.properties]: done

**Pull request closed without merging**